webserver.py

```diff
@@ -60,14 +60,23 @@ def do_GET(self):
             logger.warn("Connected to from " + str(self.client_address) + " looking at " + str(inst))
 
             with _scraped_data_lock:
-                if inst not in _scraped_data.keys():
-                    raise ValueError(str(inst) + " not known")
-                if _scraped_data[inst] == "":
-                    raise ValueError("Instrument has become unavailable")
-                try:
-                    ans = "%s(%s)" % (callback, json.dumps(_scraped_data[inst]))
-                except Exception as err:
-                    raise ValueError("Unable to convert instrument data to JSON: %s" % err.message)
+                active = {}
```

Comment:
- Preferred method for initialising Python dictionaries is `dict()`
- Incorrect scope. `active` is unused outside of the `if...`. It should be initialised within the if so that it exists only within the scope it is used